subsys/bluetooth/shell/bt.c

```diff
@@ -29,6 +30,7 @@
 
 #include <shell/shell.h>
 
+#include "../host/hci_core.h"
```

Comment:
Implement a subsys/bluetooth/shell/hci.c.  Then, #include "hci.h" from current folder.

Reply:
Not following, this is not implementing the HCI parsing just using bt_hci_cmd_send_sync or are you suggesting a wrapper?